ooniprobe/View/Informed consent/Onboarding2ViewController.m

```diff
@@ -0,0 +1,37 @@
+#import "Onboarding2ViewController.h"
+
+@interface Onboarding2ViewController ()
+
+@end
+
+@implementation Onboarding2ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.titleLabel setText:NSLocalizedString(@"what_is_ooniprobe", nil)];
+    /*
+     Constraint for iPhoneSE
+     https://stackoverflow.com/questions/43521024/how-to-set-constraints-for-iphone-5s-or-iphone-se-only-using-storyboard-in-autol
```

Comment:
Tip: you can cut everything after the number and the SO link will still work, but you'll have a shorter comment